indico/web/templates/header.html

```diff
@@ -37,11 +37,12 @@
                     <a href="{{ item.url }}">{{ item.title }}</a>
                 {% elif item.items %}
                     <a class="arrow js-dropdown" href="#" data-toggle="dropdown">{{ item.title }}</a>
-                    <ul class="dropdown">
+                    <ul class="i-dropdown">
                         {{ loop(item.items) }}
                     </ul>
                 {% endif %}
             {% endfor %}
+            <a href="/semantic-examples">Examples</a>
```

Comment:
I guess you forgot to remove that? :P

Reply:
I pushed this example commit in case anyone wanted to test it locally. I'll remove it when it's ready to be merged.

Reply:
Ok, good to know ;)